src/System.Text.RegularExpressions/tests/RegexConstructorTests.cs

```diff
@@ -9,166 +9,30 @@
-            iCountTestcases++;
-            try
-            {
-                r = new Regex("foo", RegexOptions.ECMAScript | RegexOptions.IgnoreCase | RegexOptions.Multiline | RegexOptions.CultureInvariant | RegexOptions.ExplicitCapture);
-                iCountErrors++;
-                Console.WriteLine("Err_6556jhkj Expected Regex to throw ArgumentException and nothing was thrown");
-            }
-            catch (ArgumentException)
-            {
-            }
-            catch (Exception e)
-            {
-                iCountErrors++;
-                Console.WriteLine("Err_2189jhss Expected Regex to throw ArgumentException and the following exception was thrown:\n {0}", e);
-            }
-
-            //[]RegEx with ECMA RegexOptions with all other valid options plus Singleline
-            strLoc = "Loc_9891asfes";
-            iCountTestcases++;
-            try
-            {
-                r = new Regex("foo", RegexOptions.ECMAScript | RegexOptions.IgnoreCase | RegexOptions.Multiline | RegexOptions.CultureInvariant | RegexOptions.Singleline);
-                iCountErrors++;
-                Console.WriteLine("Err_3156add Expected Regex to throw ArgumentException and nothing was thrown");
-            }
-            catch (ArgumentException)
-            {
-            }
-            catch (Exception e)
-            {
-                iCountErrors++;
-                Console.WriteLine("Err_456hjhj Expected Regex to throw ArgumentException and the following exception was thrown:\n {0}", e);
-            }
-
-            //[]RegEx with ECMA RegexOptions with all other valid options plus IgnorePatternWhitespace
-            strLoc = "Loc_23889asddf";
-            iCountTestcases++;
-            try
-            {
-                r = new Regex("foo", RegexOptions.ECMAScript | RegexOptions.IgnoreCase | RegexOptions.Multiline | RegexOptions.CultureInvariant | RegexOptions.IgnorePatternWhitespace);
-                iCountErrors++;
-                Console.WriteLine("Err_3568sdae Expected Regex to throw ArgumentException and nothing was thrown");
-            }
-            catch (ArgumentException)
-            {
-            }
-            catch (Exception e)
-            {
-                iCountErrors++;
-                Console.WriteLine("Err_4657dsacd Expected Regex to throw ArgumentException and the following exception was thrown:\n {0}", e);
-            }
-            ///////////////////////////////////////////////////////////////////
-            /////////////////////////// END TESTS /////////////////////////////
-        }
-        catch (Exception exc_general)
-        {
-            ++iCountErrors;
-            Console.WriteLine("Error Err_8888yyy!  strLoc==" + strLoc + ", exc_general==" + exc_general.ToString());
-        }
+        // Should not throw
+        new Regex("foo", RegexOptions.ECMAScript | RegexOptions.IgnoreCase | RegexOptions.Multiline | RegexOptions.CultureInvariant);
```

Comment:
Are there any properties we can assert on, e.g. to verify that what was passed in to the ctor roundtrips?